presto-main/src/main/java/io/prestosql/execution/QueryStateTimer.java

```diff
@@ -87,14 +93,15 @@ private void beginDispatching(long now)
 
     public void beginPlanning()
     {
-        beginPlanning(tickerNanos());
+        beginPlanning(tickerNanos(), currentThreadCpuTime());
     }
 
-    private void beginPlanning(long now)
+    private void beginPlanning(long now, long cpuNow)
```

Comment:
@sopel39 Thanks for the comments, what do you think of having something like this?